docs/source/generate_notebook_links.py

```diff
@@ -0,0 +1,54 @@
+from fnmatch import fnmatch
+from pathlib import Path
+from typing import List, Optional
+
+
+def create_nblink(file: Path, notebook_path: Path):
```

Comment:
I think better cast all args to Path to address issue with str if it happens

Reply:
It is a local function, not imported anywhere and it is called in the current file only once. Don't think it's necessary.